README.md

```diff
@@ -49,6 +49,14 @@ Support
 <a href="https://www.paypal.me/vishalnehra"><img width="20%" src="paypal.svg" alt="PayPal"></a>
 <a href="https://liberapay.com/Team-Amaze/donate"><img src="https://upload.wikimedia.org/wikipedia/commons/2/27/Liberapay_logo_v2_white-on-yellow.svg" alt="Liberapay" width="80px" ></a>  
 Or buy the [Cloud Plugin](https://play.google.com/store/apps/details?id=com.filemanager.amazecloud) supports Google Drive™, Dropbox, OneDrive and Box accounts.  
+Try our app - [Amaze File Utilities](https://play.google.com/store/apps/details?id=com.amaze.fileutilities)
+1. List videos / images / music documents in your device in a interactive UI where you're able to group / sort and quickly jump to any headers.
+2. Open videos / images / music / documents (pdf / docx / epub) with inbuilt player.
+3. Share / delete / cast on your tv
+4. Analyse internal storage for junk files, duplicate files, large videos / old downloads / screenshots or recordings.
+5. Analyse and group images between memes, low light / blurry / selfies / group pics.
+7. Transfer files directly between two mobile devices on same wifi network using high speed peer to peer network
+8. Gesture support in image / video player, play in background, picture in picture mode, download subtitles within the player.
```

Comment:
Do you mind taking the trouble to move this to a new PR? :)

Reply:
Let it be.